webapp/src/Controller/Jury/JudgehostController.php

```diff
@@ -117,6 +119,44 @@ public function indexAction(Request $request): Response
                 'value' => 'j' . $lastJobId['jobid'],
             ];
 
+            $now = Utils::now();
+            $timings = $this->em->createQueryBuilder()
```

Comment:
Would it make sense to run this query once outside the for loop for all judgehosts?

Reply:
Done.